src/controllers/controllermanager.cpp

```diff
@@ -404,6 +426,10 @@ void ControllerManager::slotSavePresets(bool onlyActive) {
     }
 }
 
+void ControllerManager::slotKeyboardPresetChanged(ControllerPresetPointer pPreset) {
+    emit(keyboardPresetChanged(pPreset));
```

Comment:
Yeah, that was on the TODO list in my head :-) Wasn't sure though if that was possible, thanks!